### PR TITLE
flow 0.48.0

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/flow.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Flow < Formula
   head "https://github.com/facebook/flow.git"
 
   stable do
-    url "https://github.com/facebook/flow/archive/v0.47.0.tar.gz"
-    sha256 "cf4bda660731c6d0731d1193fac458f590a1313172989b4a4561f64fbcc2cc1c"
+    url "https://github.com/facebook/flow/archive/v0.48.0.tar.gz"
+    sha256 "8772896075dc4028e62720fe18a6608f278f471931b2a8fff280d0efc0fd4f29"
 
     # Remove for > 0.47.0
     # [PR] Remove 4.04 ocaml version upper bound
```
Comment (JCount) [inline Formula/flow.rb]: Ahem

Comment (ilovezfs) [inline Formula/flow.rb]: indeed

Comment (JCount) [inline Formula/flow.rb]: 👾

**Comment by ilovezfs** [comment]
@gabelevi looks like there's a build failure.

**Comment by gabelevi** [comment]
@ilovezfs Thanks for updating the formula! I really appreciate the help!

**Comment by ilovezfs** [comment]
@gabelevi you're welcome! Thanks for the rapid upstream fix.

**Pull request closed without merging**